Formula/openjdk.rb

```diff
@@ -1,22 +1,13 @@
 class Openjdk < Formula
   desc "Development kit for the Java programming language"
   homepage "https://openjdk.java.net/"
-  if Hardware::CPU.arm?
-    # Temporarily use a openjdk 17 preview on Apple Silicon
-    # (because it is better than nothing)
-    url "https://github.com/openjdk/jdk/archive/refs/tags/jdk-17+31.tar.gz"
-    sha256 "9a658a42b2fe3b64ef3b2617395fc8f442f046e43e52b1d3b3a6a9b83d32b2ce"
-    version "16.0.2"
-  else
-    url "https://github.com/openjdk/jdk16u/archive/refs/tags/jdk-16.0.2-ga.tar.gz"
-    sha256 "d1b01bb5e710a973256a11fe852b7e23523ca8ef04997fa29cf459ba5303a476"
-  end
+  url "https://github.com/openjdk/jdk/archive/jdk-17-ga.tar.gz"
```

Comment:
Currently, versioned repo doesn't have GA tag https://github.com/openjdk/jdk17u/releases. Can consider using that if it is released there.

Reply:
As https://github.com/openjdk/jdk17u/archive/jdk-17-ga.tar.gz was released, I will update to this URL once CI finishes.